pom.xml

```diff
@@ -9,24 +9,22 @@
   
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.bdgenomics.adam</groupId>
-  <artifactId>adam-parent</artifactId>
+  <artifactId>adam-parent_2.10</artifactId>
   <version>0.16.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>ADAM</name>
+  <name>ADAM_2.10</name>
   <description>A fast, scalable genome analysis system</description>
   <url>http://bdgenomics.org/</url>
 
   <properties>
     <java.version>1.7</java.version>
-    <scala.version>2.10.4</scala.version>
```

Comment:
It's part of the dual-build process. Specifically, we "flatten" the POM by hardcoding defaults to Scala 2.10 (scala.version = 2.10.4, scala.artifact.suffix = 2.10). Then, when we want to build for Scala 2.11, we run the `scripts/move_to_scala_2.11.sh` script which just uses `sed` to swap 2.10.4 with 2.11.4 and 2.10 with 2.11. Not beautiful, but gets the job done.

If Maven allowed you to put variables in the artifact names, you wouldn't need to do this. However, as I found via experimentation, there's "good" reasons that Maven doesn't allow you to have variables in artifact names. If you required all published POM's to be "flattened", then variables in an unpublished POM would be OK.